Sources/PackageDescription4/Product.swift

```diff
@@ -130,12 +143,13 @@ public class Product: Encodable {
         }
     }
 
-    /// Create a library product that can be used by clients that depend on this package.
+    /// Create a library product to allow clients that declare a dependency on this package
+    /// to use the package’s functionality.
     ///
-    /// A library's product can either be statically or dynamically linked. It
-    /// is recommended to not declare the type of library explicitly to let the
-    /// Swift Package Manager choose between static or dynamic linking depending
-    /// on the consumer of the package.
+    /// A library’s product can either be statically or dynamically linked.
```

Comment:
the quote seems weird, shouldn't it be `'`?

Reply:
Occam uses curly quotes, I can change all of them to straight quotes if you prefer that?

Reply:
I am fine with whichever one is preferred but we should make it consistent throughout the docs.

Reply:
I decided to use the regular quotes for consistency with other docs.